### PR TITLE
Applying some general freedesktop conventions

### PR DESCRIPTION
**Pull request by thopiekar**
As needed for great flatpak integration.

### CODE REVIEW

File: deploy/org.mavlink.qgroundcontrol.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">org.mavlink.qgroundcontrol.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://docs.qgroundcontrol.com/assets/quickstart/ConnectedVehicle.jpg</image>
+    </screenshot>
+  </screenshots>
+..
```
Comment (patrickelectric) [inline deploy/org.mavlink.qgroundcontrol.metainfo.xml:30]: This two dots should be here ?

Comment (thopiekar) [reply deploy/org.mavlink.qgroundcontrol.metainfo.xml:30]: Heck?! Let me correct this.

Comment (thopiekar) [reply deploy/org.mavlink.qgroundcontrol.metainfo.xml:30]: Done. Like your reviews @patrickelectric - you have eagle eyes :stuck_out_tongue:

**Review by patrickelectric — APPROVED** [review]

File: deploy/org.mavlink.qgroundcontrol.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mavlink.qgroundcontrol</id>
+  
+  <name>QGroundControl</name>
+  <summary>UAS ground control station</summary>
+  
+  <metadata_license>MIT</metadata_license>
```
Comment (thopiekar) [inline deploy/org.mavlink.qgroundcontrol.metainfo.xml:8]: Maybe this needs discussion still. Within the generator website, I couldn't choose either GPL and/or Apache. :confused:

File: deploy/org.mavlink.qgroundcontrol.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  <project_license>GPL-3.0-only</project_license>
```
Comment (thopiekar) [inline deploy/org.mavlink.qgroundcontrol.metainfo.xml:9]: And here only one of the licenses. Don't know whether mixes are possible within these metadata files.

**Review by thopiekar — COMMENTED** [review]
Got some license questions open still.

File: deploy/org.mavlink.qgroundcontrol.metainfo.xml (hunk 3 above)
Comment (thopiekar) [reply deploy/org.mavlink.qgroundcontrol.metainfo.xml]: Found a solution to declare GPL3 and Apache2.

File: deploy/org.mavlink.qgroundcontrol.metainfo.xml (hunk 2 above)
Comment (thopiekar) [reply deploy/org.mavlink.qgroundcontrol.metainfo.xml]: Found a solution to declare GPL3 and Apache2.

**Review by LorenzMeier — APPROVED** [review]

**Comment by mrpollo** [comment]
@thopiekar can you please help us with the instructions on how to build and deploy a flatpak bundle? we need this to be documented on the QGC docs

**Comment by patrickelectric** [comment]
@thopiekar, can also be helpful to have a cmake command to generate as output the flatpak itself, this will make the documentation simple and straightforward for everybody.

**Comment by thopiekar** [comment]
@mrpollo Will do so. But need #9585 first. After that I'll create a PR with the flatpak manifest. However, without this PR building the flatpak will only be possible without airmap since I can't download anything during build-time.

@patrickelectric That shouldn't be difficult. Nevertheless, I'll keep with the plans discussed with @DonLakeFlyer and keep focusing on qmake.
Something I mentioned on Slack already: Does any of you have an overview about the current status of the CMake files? Do we build with the Airmap SDK there, for example? Any other differences. I think we should collect these things somewhere. :)

**Pull request merged**